src/Badges.sol

```diff
@@ -424,7 +422,7 @@ contract Badges is
     );
 
     // this authenticates that the claimant (leaf) is indeed part of the tree whose root was signed
-    bytes32 leaf = keccak256(abi.encodePacked(_to)); //todo:: why not use abi.encode? which is appropriate?
+    bytes32 leaf = keccak256(abi.encodePacked(_to));
```

Comment:
@rahulrumalla I looked into this. From what I read, as long as you're only encoding one argument, this is the way to go, uses slightly less gas.